src/ZuriAC.php

```diff
@@ -123,8 +123,13 @@ public function onLoad() : void {
 		self::$instance = $this;
 		ConfigManager::checkConfig();
 
+                if (!version_compare(PHP_VERSION, '8.3.0', '>=')) {
```

Comment:
```suggestion
                if (version_compare(PHP_VERSION, '8.3.0', '<')) {
```

Why does using the negation operator make the code harder to read?